apps/src/templates/Dialog.story.jsx

```diff
@@ -11,136 +11,117 @@ import Dialog, {
 import ExampleDialogButton from '../util/ExampleDialogButton';
 import {action} from '@storybook/addon-actions';
 
-export default storybook => {
-  return storybook.storiesOf('Dialogs/Dialog', module).addStoryTable([
-    {
-      name: 'basic example',
-      description: `The most common scenario is to create a straightforward dialog
-                        with an icon, a title, some body content, and a cancel/confirm
-                        button. You can do this easily by simply passing the relevant props`,
-      story: () => (
-        <Dialog
-          hideBackdrop={true}
-          icon="https://studio.code.org/blockly/media/skins/flappy/static_avatar.png"
-          title="Puzzle 3 of 10"
-          cancelText="Go Back"
-          body='The "when run" block allows you to run code when your game starts.
+// There are so many different variations that making a template is not helpful
+export default {
+  title: 'Dialog',
+  component: Dialog
+};
+
+/*
```

Comment:
I tried to preserve the instructions in a comment, so that a developer would still be able to read them!